Cargo.toml

```diff
@@ -12,6 +12,7 @@ nom = "^3.2"
 url = "1.5.1"
 futures = "0.1"
 git2 = "^0.6"
+dialoguer = "^0.1"
```

Comment:
ther is already a dependency on `rprompt` which seem to serve the same purpose. Can we only keep one?

Reply:
Rprompt does not offer the possibility of lauching `$EDITOR`. Maybe I should try to convert `rprompt` calls to `dialoguer` instead ?